.github/workflows/WebAutomatedTests.yml

```diff
@@ -51,7 +51,7 @@ jobs:
                   cache-name: cache-node-modules
               with:
                   path: ~/.npm
-                  key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+                  key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('package-lock.json') }}
```

Comment:
Why this change?

Reply:
Probably because of the Windows. I'm right?

Reply:
Sorry, I forgot to add in the description, my bad. That was related to an update in the GH runner which started to affect this action, it was suppressed as a warning and now they started to show the error. That error happens because we don't have permission to search for any package-lock file in the entire repo when using this function hashFiles.